Notifications/notifications_actions_customization.md

```diff
@@ -12,48 +12,51 @@ This document is a starting point for engaging the community and standards bodie
 
 ## Proposed Solution
 
-We propose an extension to the current Notifications API where the `NotificationAction` dictionary would have a new `style` variable of type `NotificationActionStyle`, which is an enum with 3 values: 
-- `"default"`,
-- `"success"`, and
-- `"critical"`.
-
-These styles are platform-dependent but should be distinct from each other to allow the user to easily identify its options. For example, the `"success"` action button could have a green theme - e.g., have green background color or have a green font over the default background - while the `"critical"` could be red-themed, but it is up to the platform to define the style. If the selected style is not supported by the platform, it should fallback to `"default"`
+We propose the creation of a new property `scenario` in the `NotificationOptions` dictionary of type `NotificationScenario`, that is an `enum` with 2 values: `"default"` and `"calling"`. The color treatment and ringtone capabilities will only be available in the `"calling"` scenario.
 
 ```javascript
-enum NotificationActionStyle {
+enum NotificationScenario {
   "default",
-  "success",
-  "critical"
+  "calling"
 }
 
-dictionary NotificationAction {
-  required DOMString action;
-  required DOMString title;
-  USVString icon;
-  NotificationActionStyle style = "default";
-};
+dictionary NotificationOptions {
+  NotificationScenario scenario = "default";
+  ...
+}
 ```
 
-Similarly, to support this scenario we also propose the creation of a new NotificationSoundType enum and the inclusion of a `sound` attribute to the Notification interface and a correspondent property to the NotificationOptions dictionary:
+When the scenario is `"calling"`, the notification will always have a default dismiss button, which must occupy the rightmost action button position. This way, when a `"calling"` notification is displayed to the user, we can guarantee that he or she will always be capable to dismiss it. 
+
+### Color Treatment
+
+For notifications with `"scenario"` of type `"calling"`, the User Agent (UA) should change the notification's action buttons' color in a way that the style for the dismiss button is different from the other buttons, which are specified each as an element of the `Notfication.actions` array. If the platform does not allow color treatment for notification buttons, the UA should fallback to the `"default"` scenario colors.
+
+We propose that the default dismiss action button should have a red theme - e.g., red background color with some predefined icon on it - while all the other buttons are green-themed, but it is up to the platform to define the style.
+
+### Ringtone
+
+Similarly, to support the `"calling"` scenario we also propose the creation of a new NotificationSoundType `enum` and the inclusion of a `sound` attribute to the Notification interface and a correspondent property to the NotificationOptions dictionary. If the scenario is not `"calling"` this property value must be ignored.
```

Comment:
Do we plan to eliminate the sound property in notification options and just automatically play the looping ringtone with scenario="incoming-call"?

Reply:
Forgot about removing the `sound` property. Removed it. Thanks!